esmvaltool/preprocessor/_derive.py

```diff
@@ -43,11 +43,9 @@ def get_required(short_name, field=None):
             ('tro3', 'T3' + frequency),
             ('ps', 'T2' + frequency + 's'),
         ],
-        'rtnt': [
-            ('rsdt', 'T2' + frequency + 's'),
-            ('rsut', 'T2' + frequency + 's'),
-            ('rlut', 'T2' + frequency + 's')
-        ],
+        'rtnt': [('rsdt', 'T2' + frequency + 's'),
+                 ('rsut', 'T2' + frequency + 's'), ('rlut',
+                                                    'T2' + frequency + 's')],
```

Comment:
Linebreak looked nice in the previous version.

Reply:
sticking to the almighty ```yapf```, man, you never know when Codacy decides to not like it anymore (the linebreak I mean)

Reply:
Alright, then. :zipper_mouth_face:

Reply:
Add a `,` after the last `)` and yapf will format it nicely again.